src/actions.ts

```diff
@@ -85,6 +76,7 @@ class ActionManager implements Disposable {
             await new Promise((resolve) => setTimeout(resolve, ms));
             return "ok";
         });
+        this.add("eval", (code: string, args: any) => eval_for_client(code, args));
```

Comment:
(off topic for this PR) `ActionManager` is just a wrapper around `registerCommand`.  Assuming it is still needed, it should be renamed to `Commands`. Introducing new jargon just makes things confusing.